biogeophys/FatesHydroWTFMod.F90

```diff
@@ -450,6 +482,8 @@ function th_from_psi_vg(this,psi) result(th)
 
 
     end if
+    th = max(th,this%th_res)
```

Comment:
Since @JunyanDing is re-factoring the VG schemes (see #737 ), could we remove this from this PR and re-add it in another PR coordinated with @JunyanDing

Reply:
Yes, I agree with @rgknox , this change seems a little redundant.

Reply:
Yes.